client/state/analytics/README.md

```diff
@@ -84,6 +84,30 @@ const mapDispatchToProps = {
 `loadTrackingTool :: String -> String -> Object`<br />
 `loadTrackingTool ( trackingTool )`: Creates an action to load a tracking tool that gets attached to the DOM on component mount. Pass tracking tool reference through to middleware and manage loading scripts there.
 
+### Preoprty Enhancers
+
+The above Redux actions denote the analytics data object to be sent to Tracks. Properties to be reported often need to be passed to the actions as arguments. There are times when it would be convenient to enhance the analytics data object with universal properties that are applicable to all sorts of tracking calls. This is when Enhancers come in handy. To enhance any analytics Redux action with additional property:
+
+```
+const recorder = withEnhancer( recordPageView, [ enhancer1, enhancer2, enhancer3,... ] );
+```
+
+#### [enhanceWithSiteType](https://github.com/Automattic/wp-calypso/tree/trunk/client/state/analytics/actions/enhance-with-site-type.js)
+- Enhances any analytics Redux action
+- Adds the property `site_type` specifying the selected site's type: `jetpack` or `wpcom`
+- `site_type` is captured by default when tracking calls are handled by [`<PageViewTracker>`](https://github.com/Automattic/wp-calypso/blob/trunk/client/lib/analytics/page-view-tracker/index.jsx#L116)
+- To include the `site_type` property in any other tracking calls, the respective analytics Redux action must be accomponied by `enhanceWithSiteType` ([example](https://github.com/Automattic/wp-calypso/blob/trunk/client/login/magic-login/index.jsx#L167))
```

Comment:
When providing references to code, we shouldn't be using `trunk`, because that's likely to change and the reference will no longer be valid. Instead, use the latest commit on that file, for example:

```suggestion
- To include the `site_type` property in any other tracking calls, the respective analytics Redux action must be accomponied by `enhanceWithSiteType` ([example](https://github.com/Automattic/wp-calypso/blob/9fb7ac149e55361c0c3a9639ad0caa5e15fc88b0/client/login/magic-login/index.jsx#L167))
```

There's a few more of these in the PR that need an update.